index.html

```diff
@@ -33,9 +33,9 @@
     </section> <!-- end section 1 -->
 
     <!-- second section (headline) -->
-    <section>
+    <section class="section2">
       <main class="headline">
-        <div>
+        <div class="bg-image">
           <img src="images/bg.png" alt="burger-menu" class="bg-image">
         </div>
         <h1>Hey There. <br> I'm Mohamed</h1>
```

Comment:
- [required] Kindly, your font is different from Figman as your previous code reviewer suggested👌👌, you can find your Figma font from [Google font](https://fonts.google.com/) and import it either in your `html` or `css` to use like other fonts.😍😍